pkg/util/util.go

```diff
@@ -1544,29 +1544,37 @@ func GetEnvWithDefault(key string, defaultval string) string {
 	return val
 }
 
+//IsInvalidKubeConfigError checks if specified error is due to invalid kubeconfig
+func IsInvalidKubeConfigError(err error) error {
+	if strings.Contains(err.Error(), "invalid configuration") {
+		return fmt.Errorf("invalid KUBECONFIG provided. Please point to a valid KUBECONFIG. You do not have to be logged in %w", err)
+	}
+	return nil
+}
+
 //IsValidKubeConfigPath checks if specified `KUBECONFIG` value is a valid file i.e the path should exist and be a file
 //if env `KUBECONFIG` is set then that is used or default `KUBECONFIG` is checked
-func IsValidKubeConfigPath() bool {
-	v := os.Getenv("KUBECONFIG")
-	if v == "" {
-		if home := homedir.HomeDir(); home != "" {
-			v = filepath.Join(home, ".kube", "config")
-			klog.V(4).Infof("using default kubeconfig path %s", v)
-		} else {
-			klog.V(4).Infof("no KUBECONFIG provided and cannot fallback to default")
-			return false
+func IsValidKubeConfigPath() error {
```

Comment:
I'm not sure if the return value of this function should be `error`. Function name indicates that the response to expect from it should be either Yes or No, that is, a boolean value.

Reply:
Renaming

Reply:
I'm more inclined at not changing the signature of the function, if possible.